dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventRequestParamsMapper.java

```diff
@@ -88,6 +92,11 @@ public EventOperationParams map(RequestParams requestParams) throws BadRequestEx
             "event", requestParams.getEvent(), "events", requestParams.getEvents());
 
     validateFilter(requestParams.getFilter(), eventUids);
+    Map<String, List<QueryFilter>> dataElementFilters = new HashMap<>();
+    parseFilters(dataElementFilters, requestParams.getFilter());
```

Comment:
Is there a particular reason why we are populating the Map as a side effect instead of returning it?